adapters/Vungle/VungleAdapter/GADMAdapterVungleConstants.h

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-static NSString *const _Nonnull kGADMAdapterVungleVersion = @"6.5.3.0";
+static NSString *const _Nonnull kGADMAdapterVungleVersion = @"6.6.0.0";
```

Comment:
iOS will not ship 6.6.0, we're working currently on 6.7.0 - I'd update the version accordingly.